BP/common/landed_titles/BP_e_byzantium.txt

```diff
@@ -132,73 +132,6 @@ e_byzantium = {
 					color2 = { 255 255 255 }
 				}
 			}
-			c_awjila = {
```

Comment:
Why are these getting removed here?

Reply:
Awjila was moved to Tripolitania; in WtWSMS, c_awjila is part of k_nasamoni instead of k_aegyptus. Since Rome will have access to de jure county wars, this will prevent c_awjila from being crushed by Rome quickly.